util/expansion_test.go

```diff
@@ -73,13 +73,19 @@ func TestExpansions(t *testing.T) {
 				"key2": "val2",
 			})
 
-			err := expansions.UpdateFromYaml(filepath.Join(
+			keys, err := expansions.UpdateFromYaml(filepath.Join(
 				getDirectoryOfFile(), "testdata", "expansions.yml"))
 			So(err, ShouldBeNil)
 			So(expansions.Get("marge"), ShouldEqual, "1")
 			So(expansions.Get("lisa"), ShouldEqual, "2")
 			So(expansions.Get("bart"), ShouldEqual, "3")
+			So(expansions.Get("homer"), ShouldEqual, "4")
 			So(expansions.Get("key1"), ShouldEqual, "blah")
+			So(keys, ShouldContain, "marge")
+			So(keys, ShouldContain, "lisa")
+			So(keys, ShouldContain, "bart")
+			So(keys, ShouldContain, "homer")
+			So(keys, ShouldContain, "key1")
```

Comment:
Small suggestion, but rather than change the return value of `UpdateFromYaml` to list out the new keys, wouldn't it be equivalent to just check if `expansions` contains the key? Like `So(expansions, ShouldContain, "marge")`

Reply:
This was more so testing that the keys are properly returned- the keys are used inside of `expansions.update` so that after updating the yaml, it knows what to redact if the user has `redact_file_expansions` set to true.